modules/core/src/main/scala/eu/timepit/scalasteward/scalafix/Migration.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 scala-steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.timepit.scalasteward.scalafix
+
+import cats.data.NonEmptyList
+
+final case class Migration(
+    groupId: String,
+    artifactIds: NonEmptyList[String],
+    newVersion: String,
```

Comment:
I'd put back the from version here. If someone is multiple versions behind they may need to take multiple passes to get up to date. For example a cats 0.8 -> 0.9 rule could be applied, then a 0.9 -> 1.0, but 0.8 -> 1.0 in one step may fail.

Reply:
Maybe minVersion?

Reply:
@ShaneDelmore That makes sense. Do scalafix rewrites specify somewhere what the minimal version is a rewrite can be applied to? Or is this implicit or sometimes encoded in the name of the rewrite? From the `fs2/v1` rewrite I can only assume that it will migrate code to version 1.0.0 of the library but not what the ranges of versions is at which I can safely apply this rewrite.

Reply:
No scalafix itself is not aware of the version. I think it's a good idea to specify the *from* version in this case since it's going to be automatic. Running it on a previous version won't do much damage, but probably the fix won't be complete either.